app/assets/javascripts/components/course/course.jsx

```diff
@@ -42,9 +42,20 @@ export const Course = withRouter(createReactClass({
     persistCourse: PropTypes.func.isRequired,
     fetchTimeline: PropTypes.func.isRequired
   },
+  getInitialState() {
+    return { navBarHeight: 0, courseAlertsHeight: 0 };
+  },
 
   // Fetch all the data needed to render a course page
   componentDidMount() {
+    this.timer = setTimeout(
```

Comment:
So we're waiting 1 second before checking the heights, because an alert might not be rendered yet when this component first mounts?

Reply:
Yes on initial mounts that's null